src/libraries/adaptive-curve/ConstantsLib.sol

```diff
@@ -15,11 +19,11 @@ library ConstantsLib {
     int256 public constant TARGET_UTILIZATION = 0.9 ether;
 
     /// @notice Initial rate at target per second (scaled by WAD).
-    int256 public constant INITIAL_RATE_AT_TARGET = int256(0.04 ether) / 365 days;
+    int256 public constant INITIAL_RATE_AT_TARGET = 4_00 * ONE_BPS / 365 days;
 
-    /// @notice Mininimum rate at target per second (scaled by WAD) (0.1% APR).
-    int256 public constant MIN_RATE_AT_TARGET = int256(0.001 ether) / 365 days;
+    /// @notice Minimum rate at target per second (scaled by WAD) (min APR is MAX_RATE_AT_TARGET / CURVE_STEEPNESS).
+    int256 public constant MIN_RATE_AT_TARGET = 10 * ONE_BPS / 365 days;
 
-    /// @notice Maximum rate at target per second (scaled by WAD) (1B% APR).
-    int256 public constant MAX_RATE_AT_TARGET = int256(0.01e9 ether) / 365 days;
+    /// @notice Maximum rate at target per second (scaled by WAD) (max APR is MAX_RATE_AT_TARGET * CURVE_STEEPNESS).
```

Comment:
The actual max APR could still be here for clarity (200% APR)

Reply:
even if it's 800% APR in this case though

Reply:
@MathisGD prefers to have the curve steepness
i also prefer having the max APR written clearly
we need to settle this because there is a lot of back and forth

Reply:
we can have both no?

Reply:
let's write the max rate in the comments and keep the code like that,

Reply:
isn't what it is suggested?